mp/src/game/server/momentum/mapzones_edit.h

```diff
@@ -4,60 +4,61 @@
 #pragma once
 #endif
 
-#define BUILDSTAGE_NONE         0
-#define BUILDSTAGE_START        1
-//#define BUILDSTAGE_ROTATE     2
-#define BUILDSTAGE_END          2
-#define BUILDSTAGE_HEIGHT       3
-
+#include "mapzones_build.h"
 #include "mapzones.h"
+#include "mom_player.h"
```

Comment:
Can the momentum player be forward-declared here, with the header in the implementation file?

Reply:
I'm fairly sure you can. I'll change a few things tomorrow.